apps/dashboard/app/views/batch_connect/sessions/card/_host.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if session.ssh_to_compute_node? %>
+  <p>
+    <strong><%= t('dashboard.batch_connect_sessions_stats_host') %></strong>  
+    <%= session.connect.host ? link_to(
+        session.connect.host, OodAppkit.shell.url(
+        host: session.connect.host).to_s, target: "_blank", 
+        class: "btn btn-primary btn-sm fas fa-terminal"
+       ) : t('dashboard.batch_connect_sessions_stats_undetermined_host') %>
```

Comment:
Can you turn this ternary into an if else block. It's kinda hard to read.

Plus - IDK if we want `link_to`, instead I think you can just make an `<a>` tag with right `href`, `class` and `target` attributes.

Reply:
Will work on this now